legal-api/tests/unit/api/v1/test_business_director.py

```diff
@@ -34,7 +34,9 @@ def test_get_business_directors(session, client, jwt):
     officer = {
         'firstName': 'Michael',
         'lastName': 'Crane',
-        'middleInitial': 'Joe'
+        'middleInitial': 'Joe',
+        'partyType': 'person',
+        'orgName': ''
```

Comment:
Does this mean that `orgName` is now a required property?

Does the schema need to be updated?

Reply:
No, it is required only for the tests as part of the `officer` dict.